app/pages/sections/supplier/supplier/controller.js

```diff
@@ -27,11 +27,31 @@ const formatPutData = data => ({
 });
 
 export const getSupplierPageContext = async ({ orderId, supplierId, accessToken }) => {
-  const getSupplierDataEndpoint = getEndpoint({ endpointLocator: 'getSupplier', options: { supplierId } });
-  const supplierData = await getData({ endpoint: getSupplierDataEndpoint, accessToken, logger });
+  const ordapiSupplierDataEndpoint = getEndpoint({ endpointLocator: 'getOrdapiSupplier', options: { orderId } });
+  const ordapiSupplierData = await getData({
+    endpoint: ordapiSupplierDataEndpoint, accessToken, logger,
+  });
 
-  const context = getContext({ orderId, supplierData });
-  return context;
+  if (ordapiSupplierData.name) {
```

Comment:
can we have a double guard here `ordapiSupplierData && ordapiSupplierData.name`

Reply:
yeah I saw this on yours